tests/core/views/test_api.py

```diff
@@ -25,10 +25,40 @@
 WALLET = get_wallet_class()
 
 
-# check if the client is working
+# create account POST /api/v1/account
 @pytest.mark.asyncio
-async def test_core_views_generic(client):
-    response = await client.get("/")
+async def test_create_account(client):
+    response = await client.post("/api/v1/account", json={"name": "test"})
+    assert response.status_code == 200
+    result = response.json()
+    assert "name" in result
+    assert result["name"] == "test"
+    assert "balance_msat" in result
+    assert "id" in result
+    assert "user" in result
```

Comment:
- add test for `patch` (include currency update)

Reply:
TODO: add test for patch